feat(network): add backwards compatibility to tracker for TN3 status

harbu opened the pull request:
Add backwards compatibility to tracker for Testnet-3 nodes status
messages. Should be removed when Brubeck is launched and testnet nodes
are no longer present (at least in large amounts).

mirpo commented on src/connection/WsEndpoint.js:
```diff
@@ -21,58 +24,43 @@ function transformToObjectWithLowerCaseKeys(o) {
         }
-        if (!customHeaders) {
-            throw new Error('customHeaders not given')
+        if (!peerInfo) {
```
Maybe it's better instanceOf `PeerInfo` ?

harbu replied:
fixed

mirpo commented on src/connection/WsEndpoint.js:
```diff
@@ -213,13 +204,14 @@ class WsEndpoint extends EventEmitter {
         this.metrics.inc('onReceive')
-        this.debug('received from %s message "%s"', address, message)
-        this.emit(events.MESSAGE_RECEIVED, address, message)
+        this.debug('received from %s / %s message "%s"', peerInfo, address, message)
```
`/` or `,` ?

harbu replied:
I went with square brackets now. So:

```
received from node-2 (node) [ws://127.0.0.1:30953] message "{"version":"17.3.0","code":0,"payload":{"streams":{"stream-1::0":{"inboundNodes":[],"outboundNodes":[]}},"started":"2/10/2020, 4:27:04 PM"}}"
```

mirpo commented on src/logic/Tracker.js:
```diff
@@ -18,7 +17,6 @@ module.exports = class Tracker extends EventEmitter {
 
         // set default options
         const defaultOptions = {
```
do we need this almost empty `defaultOptions`?

harbu replied:
fixed

mirpo commented on src/protocol/NodeToNode.js:
```diff
@@ -59,40 +55,35 @@ class NodeToNode extends EventEmitter {
-    onPeerConnected(peerId) {
-        if (this.basicProtocol.peerBook.isNode(peerId)) {
-            this.emit(events.NODE_CONNECTED, peerId)
+    onPeerConnected(peerInfo) {
```
🥇

harbu committed:
feat(network): add backwards compatibility to tracker for TN3 status

Add backwards compatibility to tracker for Testnet-3 nodes status
messages. Should be removed when Brubeck is launched and testnet nodes
are no longer present (at least in large amounts).
